### PR TITLE
Push to production only on production branch release

### PR DESCRIPTION
**Pull request by reza-fs**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -174,6 +174,4 @@ workflows:
           context: binary-frontend-artifact-upload
           filters:
             branches:
-              ignore: /.*/
-            tags:
-              only: /^production.*/
+              only: /^production$/
```
Comment (afshinpaydar-binary) [inline .circleci/config.yml:177]: We're using Git tags for image versioning and without tag this workflows will fail. Also it's best practice that we have specific and meaningful version related to each FrontEnd app release/docker image/k8s release.  As I said we're using Github Tag for this purpose for now.

Comment (reza-fs) [reply .circleci/config.yml:177]: But this is not the case for https://github.com/binary-com/websockets/blob/gh-pages/.circleci/config.yml, which we try to replicate. What is your suggestion on the proposed automation on this?

Comment (afshinpaydar-binary) [reply .circleci/config.yml:177]: That Circleci is not ideal, we used [CIRCLE_SHA1](https://github.com/binary-com/websockets/blob/1a91f28a674219dc3efc5c25f0246a33bd6c0b5c/.circleci/config.yml#L122) there, But if you insist to copy from that project please change these two lines ([1](https://github.com/binary-com/deriv-developers-portal/blob/f607d6651ab8433c231f43f1f40488607e09f7a8/.circleci/config.yml#L154), [2](https://github.com/binary-com/deriv-developers-portal/blob/f607d6651ab8433c231f43f1f40488607e09f7a8/.circleci/config.yml#L157)) from `CIRCLE_TAG ` to `CIRCLE_SHA1 `

Comment (reza-fs) [reply .circleci/config.yml:177]: Thanks for the input, I've made the changes.

**Review by afshinpaydar-binary — APPROVED** [review]

**Pull request merged**